docs/conversion/sakai_20_2-20_3_oracle_conversion.sql

```diff
@@ -19,3 +19,7 @@ ALTER TABLE lti_tools RENAME COLUMN allowsettings TO allowsettings_ext;
 -- UPDATE lti_tools SET allowsettings_ext=allowsettings;
 -- ALTER TABLE lti_tools DROP COLUMN allowsettings;
 -- END SAK-44810
+
+-- SAK-41502
+ALTER TABLE GB_GRADING_EVENT_T ADD IS_EXCLUDED NUMBER(10, 0);
```

Comment:
The original PR is slightly different:

`ALTER TABLE GB_GRADING_EVENT_T ADD IS_EXCLUDED NUMBER(1,0) DEFAULT 0 NOT NULL;`

Reply:
SAK-41502 was never merged to 20.x until yesterday, which adds the column, I would say the 20.0 script is wrong.
And I think it's been modified from the original.

Reply:
I've updated to remove the 20.0 conversion

Reply:
I agree that it should only be in 1 script, not 2. However, I'm not convinced that the old ones were "wrong". Can you elaborate? I specifically asked for this column to be not-nullable and have a proper default in the sakaiproject/sakai PR almost a year ago.

Reply:
Do we really need this column to be able to store numbers up to 10 billion? That seems excessive.